### PR TITLE
Stuff

### PR DESCRIPTION
**Pull request by UnderscoreRayless**
woah rayless pr!! the best prs

### CODE REVIEW

File: crashes.json [hunk 1]
```diff
@@ -1162,6 +1162,68 @@
...
+      "fixtype": 2,
+      "fix": "Consider using a different blockoverlay and removing your current one.",
+      "causes": [
+        {
+          "method": "contains",
+          "value": "at me.aycy.blockoverlay.BlockOverlayRender.drawOverlay(BlockOverlayRender.java:65)"
+        },
+        {
+          "method": "contains",
+          "value": "at me.aycy.blockoverlay.BlockOverlayRender.onDrawBlockHighlight(BlockOverlayRender.java:33)"
```
Comment (nacrt) [inline crashes.json:1175]: try not to include the line info for future additions

File: crashes.json [hunk 2]
```diff
@@ -1162,6 +1162,68 @@
         }
       ]
     },
+    {
+      "fixtype": 2,
+      "fix": "Consider using a different blockoverlay and removing your current one.",
+      "causes": [
+        {
+          "method": "contains",
+          "value": "at me.aycy.blockoverlay.BlockOverlayRender.drawOverlay(BlockOverlayRender.java:65)"
```
Comment (nacrt) [inline crashes.json:1171]: try not to include the line info for future additions

File: crashes.json [hunk 3]
```diff
@@ -1162,6 +1162,68 @@
...
+        }
+      ]
+    },
+    {
+      "fixtype": 2,
+      "fix": "Remove BazaarNotifier from your mods folder or get support at discord.gg/wjpJSVSwvD ",
+      "causes": [
+        {
+          "method": "contains",
+          "value": "at dev.meyi.bn.handlers.ChestTickHandler.updateBazaarOrders(ChestTickHandler.java:76)"
```
Comment (nacrt) [inline crashes.json:1185]: try not to include the line info for future additions

File: crashes.json [hunk 4]
```diff
@@ -1162,6 +1162,68 @@
...
+        }
+      ]
+    },
+    {
+      "fixtype": 2,
+      "fix": "Try deleting the `screenshot-checksum-caches.json` file in `.minecraft/essential`",
```
Comment (nacrt) [inline crashes.json:1191]: use path placeholders instead

File: crashes.json [hunk 5]
```diff
@@ -1162,6 +1162,68 @@
...
+      "causes": [
+        {
+          "method": "contains",
+          "value": "gg.essential.lib.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_ARRAY but was STRING at line 1 column 1 path $"
+        }
+      ]
+    },
+    {
+      "fixtype": 2,
+      "fix": "Update to a stable version of EvergreenHUD `-mod evergreenhud`",
```
Comment (nacrt) [inline crashes.json:1201]: don't include discord commands when this is also shown ingame

File: crashes.json [hunk 6]
```diff
@@ -1162,6 +1162,68 @@
...
+      "causes": [
+        {
+          "method": "contains",
+          "value": "at dev.isxander.evergreenhud.ui.components.ElementDescriptionComponent$1.invoke(ElementDescriptionComponent.kt:66)"
```
Comment (nacrt) [inline crashes.json:1205]: try not to include the line info for future additions

File: crashes.json [hunk 7]
```diff
@@ -1162,6 +1162,68 @@
...
+        }
+      ]
+    },
+    {
+      "fixtype": 2,
+      "fix": "Remove Skyclient Updater from your blocks folder",
+      "causes": [
+        {
+          "method": "contains",
+          "value": "java.lang.NoClassDefFoundError: Could not initialize class gg.essential.config.EssentialConfig"
```
Comment (nacrt) [inline crashes.json:1219]: this crash is essentials fault
- systemless

Comment (UnderscoreRayless) [reply crashes.json:1219]: I know but it's fixed by removing scu, if essential fixes it, all the better as this crash will never happen

**Pull request merged**